src/components/app/QuickActions.js

```diff
@@ -249,11 +234,12 @@ export class QuickActions extends Component {
     resolve,
     reject
   ) {
+    const { relatedRef } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['relatedRef' is missing in props validation (react/prop-types)](https://app.codacy.com/app/metasfresh/metasfresh-webui-frontend/pullRequest?prid=4613375)